OptionSelector/CQShadesData.m

```diff
@@ -0,0 +1,220 @@
+
+-(void)initializeData{
+
+    self.red = [[CQCategory alloc] init];
+    self.yellow = [[CQCategory alloc] init];
+    self.blue = [[CQCategory alloc] init];
+    
+    self.red.name = @"Red";
+    self.yellow.name = @"Yellow";
+    self.blue.name = @"Blue";
+    
+    self.red.detailTitle = @"Shades of Red";
+    self.yellow.detailTitle = @"Shades of Yellow";
+    self.blue.detailTitle = @"Shades of Blue";
+    
+    self.red.options = @[@"cherry",
+                         @"rose",
+                         @"jam",
+                         @"merlot",
+                         @"garnet",
+                         @"crimson",
+                         @"ruby",
+                         @"scarlet",
+                         @"wine",
+                         @"brick",
+                         @"apple",
+                         @"mahogany",
+                         @"blood",
+                         @"sangria",
+                         @"berry",
+                         @"currant",
+                         @"blush",
+                         @"candy",
+                         @"lipstick"
+                         ];
+    
+    self.yellow.options = @[@"canary",
+                            @"gold",
+                            @"daffodil",
+                            @"flaxen",
+                            @"butter",
+                            @"lemon",
+                            @"mustard",
+                            @"corn",
+                            @"medallion",
+                            @"dandelion",
+                            @"fire",
+                            @"bumblebee",
+                            @"banana",
+                            @"butterscotch",
+                            @"dijon",
+                            @"honey",
+                            @"blonde",
+                            @"pineapple",
+                            @"tuscan sun"
+                            ];
+    
+    self.blue.options = @[@"slate",
+                          @"sky",
+                          @"navy",
+                          @"indigo",
+                          @"cobalt",
+                          @"teal",
+                          @"ocean",
+                          @"peacock",
+                          @"azure",
+                          @"cerulean",
+                          @"lapis",
+                          @"spruce",
+                          @"stone",
+                          @"aegean",
+                          @"blueberry",
+                          @"denim",
+                          @"admiral",
+                          @"sapphire",
+                          @"arctic"
+                          ];
+   
+    //Without (CGFLoat) OR the decimal (.0) after the number the UIColor will not show; one OR the other is sufficient.
+    UIColor *cherry = [UIColor colorWithRed:(CGFloat)164.0/255.0 green:(CGFloat)0.0/255.0 blue:(CGFloat)0.0/255.0 alpha:(CGFloat)1.0];
```

Comment:
This sounds great.  I'm not sure how I'd put that together though.  Could you please explain this a little further @calebegg ?